projects/components/src/select/select-group.component.ts

```diff
@@ -19,10 +20,13 @@ export class SelectGroupComponent implements AfterContentInit {
   @ContentChildren(SelectComponent)
   private readonly selectChildren!: QueryList<SelectComponent<unknown>>;
 
+  public constructor(private readonly subscriptionLifecycle: SubscriptionLifecycle) {}
   public ngAfterContentInit(): void {
-    queryListAndChanges$(this.selectChildren)
-      .pipe(observeOn(asyncScheduler))
-      .subscribe(list => this.setPositions(list));
+    this.subscriptionLifecycle.add(
+      queryListAndChanges$(this.selectChildren)
```

Comment:
query list is an angular component-local component. I would assume it already completes on component destroy.